src/secure_random.cr

```diff
@@ -1,19 +1,61 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom library is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others.
```

Comment:
I think it is delegated from LibCrypto instead? SecureRandom uses several methods from LibCrypto but LibCrypto does not use any method from SecureRandom.